examples/main/views/custom-interaction.js

```diff
@@ -18,11 +18,13 @@
 // OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
 // THE SOFTWARE.
 
-/* window */
 import React, {Component} from 'react';
 import PropTypes from 'prop-types';
-import MapGL, {autobind} from 'react-map-gl';
+import autobind from 'react-autobind';
+import {InteractiveMap} from 'react-map-gl';
+import DeckGL, {ArcLayer} from 'deck.gl';
```

Comment:
> Do we want to pull in DeckGL into a basic example?

Ideally not. That should probably be a special deck.gl example, or just a reference to deck.gl's examples.

What we need to make the point here is some kind of overlay component that remains visible after the map no longer displays due to pitch > 60. Maybe we can use your new markers - unless they are dependent on some map context that also disappears? 

Perhaps this example would be a good test case that all the new components support the non-visible map case?